CancerGov/_src/Scripts/NCI/UX/AppModuleSpecific/BasicCTS/Common/select2-intervention-initializer.js

```diff
@@ -18,201 +18,198 @@
 // adapter.  (If you can find a cleaner way, feel free)
 //
 
-define(function(require) {
-    var $ = require('jquery');
-    require('select2');
+var $ = require('jquery');
 
```

Comment:
The only true changes to this file is removal of the select2 module. There are some tab changes below that have thrown off this whole file comparison, but they are fake news